### PR TITLE
[tests-only] Skip tests related to issue-ocis-1328  on ocis

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR skips tests related to the issue https://github.com/owncloud/ocis/issues/1328 for OCIS as these will be implemented for spaces but not like they are implemented in OC10 now.

## Related Issue

- https://github.com/owncloud/ocis/issues/1328

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareCreateSpecialToShares1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/36349/61

File: tests/acceptance/features/apiCapabilities/capabilities.feature [hunk 1]
```diff
@@ -134,7 +134,7 @@ Feature: capabilities
       | files      | file_locking_support            | 1     |
       | files      | file_locking_enable_file_action | 1     |
 
-  @smokeTest @skipOnOcV10.6 @skipOnOcV10.7.0
+  @smokeTest @skipOnOcV10.6 @skipOnOcV10.7.0 @skipOnOcis
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiCapabilities/capabilities.feature:137]: We noticed some scenarios(like this one) that are not involving any sharing setting configurations but were skipped previously in the oCIS CI with the same issue. What should we do with such scenarios? What capabilities items are available in the oCIS? @individual-it @phil-davis

File: tests/acceptance/features/apiCapabilities/capabilities.feature [hunk 2]
```diff
@@ -287,8 +287,8 @@ Feature: capabilities
       | capability    | path_to_element | value |
       | files_sharing | can_share       | 1     |
 
-
-  #feature added in #32414 released in 10.0.10
+  @skipOnOcis
+  #feature added in #32414 released in 10.0.10 #todo
```
[the remark this answers is not in the document]
Comment (SwikritiT) [reply tests/acceptance/features/apiCapabilities/capabilities.feature:291]: The todo was for mE. I'll remove it in nice commit. Nice catch!

File: tests/acceptance/features/apiCapabilities/capabilities.feature (hunk 1 above)
Comment (phil-davis) [reply tests/acceptance/features/apiCapabilities/capabilities.feature:137]: We should skip all these.
We should add a capability test to the "lcal" API tests in oCIS that demonstrates what capabilities oCIS currently reports. That will reduce regressions of the capabilities endpoint in oCIS.

File: tests/acceptance/features/apiCapabilities/capabilities.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (phil-davis) [reply tests/acceptance/features/apiCapabilities/capabilities.feature]: fixed

**Review by phil-davis — APPROVED** [review]

**Pull request merged**